src/tree/SubscriptionTreeItem.ts

```diff
@@ -27,14 +31,32 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean): Promise<AzExtTreeItem[]> {
+
+        let treeItemPostgres: AzExtTreeItem[];
+        let treeItem: AzExtTreeItem[];
+
+        //Postgres
+        const clientPostgres: PostgreSQLManagementClient = createAzureClient(this.root, PostgreSQLManagementClient);
+        const accountsPostgres: ServerListResult = await clientPostgres.servers.list();
+        const databasesPostgres = await clientPostgres.databases.listByServer("myResourceGroup", accountsPostgres[0].name);
```

Comment:
As a general rule, you want to delay calls until you actually need the information. In this case, you don't need to list the databases until the user expands the `PostgreSQLAccountTreeItem`. Imagine if the user has hundreds of accounts and each one is listing all of it's databases here - that could take a long time even if the user never expands a single account tree item. This could should probably live in `PostgreSQLAccountTreeItem.loadMoreChildrenImpl`.